### PR TITLE
naut & gbarnz Keymaps and Readme

### PR DESCRIPTION
**Pull request by nautxx**
## Description
New keymaps added for Hasu USB to USB, Ramaworks U80 and HHKB. Revised old keymaps for Ramaworks M65-A and M6-A. Readme's updated.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* N/A

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/converter/usb_usb/keymaps/naut/config.h [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#pragma once
```
Comment (drashna) [inline keyboards/converter/usb_usb/keymaps/naut/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/converter/usb_usb/keymaps/naut/keymap.c [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/converter/usb_usb/keymaps/naut/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/hhkb/ansi/keymaps/naut/keymap.c [hunk 3]
```diff
@@ -0,0 +1,56 @@
+/*  -*-  eval: (turn-on-orgtbl); -*-
```
Comment (drashna) [inline keyboards/hhkb/ansi/keymaps/naut/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/jc65/v32a/keymaps/gbarnz/keymap.c [hunk 4]
```diff
@@ -0,0 +1,31 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/jc65/v32a/keymaps/gbarnz/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/jc65/v32a/keymaps/naut/keymap.c [hunk 5]
```diff
@@ -1,84 +1,31 @@
 #include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/jc65/v32a/keymaps/naut/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/jc65/v32u4/keymaps/naut/keymap.c [hunk 6]
```diff
@@ -1,93 +1,37 @@
 #include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/jc65/v32u4/keymaps/naut/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/wilba_tech/rama_works_m60_a/keymaps/naut/keymap.c [hunk 7]
```diff
@@ -0,0 +1,33 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/wilba_tech/rama_works_m60_a/keymaps/naut/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/wilba_tech/rama_works_m6_a/keymaps/naut/keymap.c [hunk 8]
```diff
@@ -1,17 +1,16 @@
 #include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/wilba_tech/rama_works_m6_a/keymaps/naut/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/wilba_tech/rama_works_u80_a/keymaps/naut/keymap.c [hunk 9]
```diff
@@ -0,0 +1,35 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/wilba_tech/rama_works_u80_a/keymaps/naut/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by nautxx — COMMENTED** [review]
Requested changes made.

**Review by drashna — APPROVED** [review]

File: keyboards/converter/usb_usb/keymaps/naut/config.h [hunk 10]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2021 Jason Barnachea
```
Comment (fauxpark) [inline keyboards/converter/usb_usb/keymaps/naut/config.h:1]: This file can be deleted.

File: keyboards/jc65/v32a/keymaps/gbarnz/rules.mk [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+  *
+  * This program is distributed in the hope that it will be useful,
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  * GNU General Public License for more details.
+  *
+  * You should have received a copy of the GNU General Public License
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  */
+
```
Comment (fauxpark) [inline keyboards/jc65/v32a/keymaps/gbarnz/rules.mk:16]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/hhkb/ansi/keymaps/naut/keymap.c [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  */
+
+/*  -*-  eval: (turn-on-orgtbl); -*-
+ * default HHKB Layout
+ */
+#include QMK_KEYBOARD_H
+
+#define BASE 0
+#define HHKB 1
```
Comment (drashna) [inline keyboards/hhkb/ansi/keymaps/naut/keymap.c:23]:
```suggestion
enum custom_layers {
    _BASE,
    _HHKB,
};
```

File: keyboards/hhkb/ansi/keymaps/naut/keymap.c [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+     |-------+---+---+---+---+---+---+---+---+---+---+-------+-----+-------+---|
+     | Shift | Z | X | C | V | B | N | M | , | . | / | Shift | Fn0 |       |   |
+     |-------+---+---+---+---+---+---+---+---+---+---+-------+-----+-------+---|
+
+            |------+------+-----------------------+------+------|
+            | LAlt | LGUI | ******* Space ******* | RGUI | RAlt |
+            |------+------+-----------------------+------+------|
+    */
+
+    [BASE] = LAYOUT( //  default layer
```
Comment (drashna) [inline keyboards/hhkb/ansi/keymaps/naut/keymap.c:43]:
```suggestion
    [_BASE] = LAYOUT( //  default layer
```

File: keyboards/hhkb/ansi/keymaps/naut/keymap.c [hunk 14]
```diff
@@ -0,0 +1,72 @@
...
+        KC_ESC, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_BSLS, KC_GRV,
+        KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, KC_BSPC,
+        KC_LCTL, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_ENT,
+        KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_RSFT, MO(HHKB),
```
Comment (drashna) [inline keyboards/hhkb/ansi/keymaps/naut/keymap.c:47]:
```suggestion
        KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_RSFT, MO(_HHKB),
```

File: keyboards/hhkb/ansi/keymaps/naut/keymap.c [hunk 15]
```diff
@@ -0,0 +1,72 @@
...
+      |      |     |     |     |    |    | +  | -  | End | PgD | Dow |     |       |       |     |
+      |------+-----+-----+-----+----+----+----+----+-----+-----+-----+-----+-------+-------+-----|
+
+                 |------+------+----------------------+------+------+
+                 | **** | **** | ******************** | **** | **** |
+                 |------+------+----------------------+------+------+
+
+     */
+
+    [HHKB] = LAYOUT(
```
Comment (drashna) [inline keyboards/hhkb/ansi/keymaps/naut/keymap.c:67]:
```suggestion
    [_HHKB] = LAYOUT(
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Pull request closed without merging**